packages/mui-system/src/colorManipulator/colorManipulator.test.js

```diff
@@ -275,23 +276,23 @@ describe('utils/colorManipulator', () => {
 
   describe('alpha', () => {
     it('converts an rgb color to an rgba color with the value provided', () => {
-      expect(alpha('rgb(1, 2, 3)', 0.4)).to.equal('rgba(1, 2, 3, 0.4)');
+      expect(alpha('rgb(1, 2, 3)', 0.4)).to.equal(hex('rgba(1, 2, 3, 0.4)'));
```

Comment:
All of these cases with the added `hex()` translation are due to the new runtime output as `#rrggbbaa`.